src/main/kotlin/me/partlysanestudios/partlysaneskies/utils/MiscUtils.kt

```diff
@@ -0,0 +1,75 @@
+package me.partlysanestudios.partlysaneskies.utils
```

Comment:
Su said this could all go into HypixelUtils idk

Reply:
I literally just glanced at it

Reply:
I'm just not a huge fan of misc things

Reply:
Oh